Setting up the perf CI runs in dotnet/SDK

johnbeisner opened the pull request:
@mmitche 
@dotnet/dotnet-cli

Commit:
Adding a 'dotnet/toolset' repo

Commit:
Merge remote-tracking branch 'upstream/master'

Commit:
Setting up the perf CI runs in dotnet/SDK

DrewScoggins commented on data/repolist.txt:
```diff
@@ -30,9 +30,7 @@ dotnet/cli branch=release/2.0-vs server=dotnet-ci
 dotnet/cli branch=release/15.5 server=dotnet-ci
 dotnet/cli branch=dev/release/2.0 server=dotnet-ci
 dotnet/cli branch=dev/repo-refactoring server=dotnet-ci
-dotnet/core-sdk branch=dev/release/2.0 server=dotnet-ci
```
Why are you removing these entries? Are they no longer relevant? When I looked on ci.dot.net I saw that they have not run in almost four months

johnbeisner replied:
I don't know why my master wants to remove those - I'll put them back.

Commit:
Putting back:
dotnet/core-sdk branch=dev/release/2.0 server=dotnet-ci
dotnet/toolset branch=dev/release/2.0 server=dotnet-ci

DrewScoggins commented:
LGTM
